### PR TITLE
Update turnstile

### PR DESCRIPTION
**Pull request by luluhoc**

### CODE REVIEW

**Comment by luluhoc** [comment]
@azeriker

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@zennolab_com/capmonstercloud-client",
+  "name": "@luluhoc/capmonstercloud-client-patched",
```
Comment (Bezzz23) [inline package.json:2]: Need revert to original

File: package.json [hunk 2]
```diff
@@ -10,10 +10,10 @@
   "files": [
     "dist"
   ],
-  "homepage": "https://github.com/ZennoLab/capmonstercloud-client-js",
+  "homepage": "https://github.com/luluhoc/capmonstercloud-client-js",
```
Comment (Bezzz23) [inline package.json:13]: Revert

File: package.json [hunk 3]
```diff
@@ -10,10 +10,10 @@
...
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/ZennoLab/capmonstercloud-client-js.git"
+    "url": "git+https://github.com/luluhoc/capmonstercloud-client-js.git"
```
Comment (Bezzz23) [inline package.json:16]: Revert

File: package.json [hunk 4]
```diff
@@ -26,7 +26,7 @@
     "test-unit": "jest -c jest.config.unit.js",
     "test-integr": "jest -c jest.config.integration.js"
   },
-  "author": "",
+  "author": "Lucjan Grzesik",
```
Comment (Bezzz23) [inline package.json:29]: Revert

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@zennolab_com/capmonstercloud-client",
-  "version": "1.3.0",
```
Comment (azeriker) [inline package.json:3]: need to up minor version to 1.4.0

File: package.json (hunk 1 above)
Comment (luluhoc) [reply package.json:2]: sorry forgot about it

**Review by azeriker — APPROVED** [review]

**Pull request merged**